config/locales/es.yml

```
@@ -1027,6 +1027,10 @@
       body_title:
         one: "El siguiente hallazgo se encuentra sin respuesta"
         other: "Los siguientes hallazgos están sin respuesta"
+    new_admin_user:
+      title: "Nuevo administrador asignado"
+      body_title: "Se asigno un nuevo administrador"
+      body: "Se ha promovido a %{name} como nuevo administrador"
```

No tuve mucha inspiración con los textos xD

---

Jaja, de una, le pondría (tomando en cuenta el cambio desde el template):
```suggestion
      body: "%{name} (%{user}) ahora es administrador"
```